src/oop2.py

```diff
@@ -16,7 +17,11 @@ def __init__(self, num_wheels):
 # of wheels to 2 by passing that to the constructor of its superclass.
 #
 # Override the drive() method in Motorcycle so that it prints "BRAAAP!!"
-
+class Motorcycle(GroundVehicle):
+    def __init__(self, num_wheels=2):
+        self.num_wheels = num_wheels
```

Comment:
you want to get this from Vehicle. something like`super().__init__(2)` or `GroundVehicle.__init__(self, num_wheels)`